src/common/sorted-table/index.js

```diff
@@ -134,9 +143,40 @@ class ColumnHead extends Component {
 
 // ===================================================================
 
+class Checkbox extends React.PureComponent {
+  componentDidUpdate () {
+    const { props: { indeterminate }, ref } = this
+    if (ref !== null && indeterminate !== undefined) {
+      ref.indeterminate = indeterminate
+    }
+  }
+
+  _ref = ref => {
+    this.ref = ref
+    this.componentDidUpdate()
+  }
+
+  render () {
+    const { indeterminate, ...props } = this.props
+    props.ref = this._ref
+    props.type = 'checkbox'
+    return <input {...props} />
+  }
+}
+
+// ===================================================================
+
 const DEFAULT_ITEMS_PER_PAGE = 10
+const actionsShape = propTypes.arrayOf(propTypes.shape({
+  handler: propTypes.func.isRequired,
+  icon: propTypes.string.isRequired,
+  label: propTypes.node.isRequired,
+  level: propTypes.oneOf([ 'warning', 'danger' ])
+}))
 
 @propTypes({
+  groupedActions: actionsShape,
+  individualActions: actionsShape,
```

Comment:
Keep prop types sorted.

**Rule:** when entries can be in any order, choose alphabetic order.